index.js

```diff
@@ -67,6 +67,15 @@ let cachedRecords;
 let credentials;
 
 /**
+ * Maps between passed in user name and internal user name.
+ *
+ * @type {Object}
+ *
```

Comment:
I think this is sufficient based on the assumption that within a test scenario no one is going to try to create the same user multiple times, and even if someone attempts to do it they'll get the error above 👍